src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -82,12 +82,6 @@ public bool GenerateFullPaths
             get { return _store.GetOrDefault(nameof(GenerateFullPaths), false); }
         }
 
-        public string LangVersion
```

Comment:
@jaredpar @gafter @AnthonyDGreen Are the csc and vbc tasks in scope for public API discipline?